### PR TITLE
Make GET endpoints public and add CD

### PR DESCRIPTION
**Pull request by kfriedman**
Hello,

As discussed, this Repo does a few things:

- Adds continuous deployment from Travis
- Remove authentication on the `GET` endpoints from the Swagger docs
- Fixes a hanging Mocha test
- Some StandardJS style fixes
- Removes the Lambda from our VPC (just a reminder, [Lambda Best Practices](https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html) advises to include Lambdas in VPCs only if absolutely necessary because it increases cold-start times) 

Thanks!

### CODE REVIEW

File: config/production.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+BOOK_LISTS_BUCKET=book-lists
```
Comment (nonword) [inline config/production.env]: Is this supposed to be "book-lists-production"? I think in general we're using environment suffixes on buckets

Comment (kfriedman) [reply config/production.env]: Good catch! It's fixed.

File: test/test-helper.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ before(function () {
     try {
       let localPath = path.join('./test/data/getObject', params.Key)
       let content = fs.readFileSync(localPath)
-      callback(null, { Body: new Buffer(content) })
+      callback(null, { Body: Buffer.from(content) })
```
Comment (EdwinGuzman) [inline test/test-helper.js]: Is this change better?

File: package.json [hunk 3]
```diff
@@ -4,11 +4,14 @@
...
+    "deploy-qa": "./node_modules/.bin/node-lambda deploy -e qa -f ./config/qa.env -o arn:aws:iam::946183545209:role/lambda-full-access -a $aws_access_key_id_qa -s $aws_secret_access_key_qa",
+    "deploy-qa-local": "./node_modules/.bin/node-lambda deploy -e qa -f ./config/qa.env -g sg-116eeb60 --role arn:aws:iam::946183545209:role/lambda-full-access --profile nypl-digital-dev",
+    "deploy-production": "./node_modules/.bin/node-lambda deploy -e production -f ./config/production.env -o arn:aws:iam::946183545209:role/lambda-full-access -a $aws_access_key_id_production -s $aws_secret_access_key_production",
+    "deploy-production-local": "./node_modules/.bin/node-lambda deploy -e production -f ./config/production.env -g sg-116eeb60 --role arn:aws:iam::946183545209:role/lambda-full-access --profile nypl-digital-dev",
     "run-development": "sam local start-api --port 3001 --profile nypl-sandbox --template ./sam-template.development.yml",
-    "deploy-qa": "./node_modules/.bin/node-lambda deploy -e qa -f ./config/qa.env -b subnet-59bcdd03,subnet-5deecd15 -g sg-116eeb60 --role arn:aws:iam::946183545209:role/lambda-full-access --profile nypl-digital-dev",
```
Comment (EdwinGuzman) [inline package.json]: The subnets are not needed for these deployments?

**Comment by kfriedman** [comment]
Oh! I fixed the `book-lists-development` bucket name. I wasn't aware there was a development bucket with that name. Thanks @nonword!

File: test/test-helper.js (hunk 2 above)
Comment (kfriedman) [reply test/test-helper.js]: StandardJS linter was giving me an error. I think the reasons are here:
https://github.com/standard/standard/issues/693

Comment (EdwinGuzman) [reply test/test-helper.js]: Great, that makes sense. Thanks.

File: package.json (hunk 3 above)
Comment (kfriedman) [reply package.json]: Yeah, since there's no database access or other reasons to be on our VPC, it's best practice to not put Lambdas on a VPC.

**Pull request merged**